src/cluster/cluster_test.go

```diff
@@ -37,7 +37,7 @@ func (s *ClusterSuite) SetupSuite() {
 	)
 }
 
-func (s *ClusterSuite) GenerateDevices(count int) []*model.Device {
+func GenerateDevices(count int) []*model.Device {
```

Comment:
Whats the thinking around removing this from the test suite struct?

Reply:
It's not a test and it didn't need access to the suite members. I should probably have also made it private to the file at the same time. 
I don't feel strongly about this fwiw, happy to put it back if you do.